sources/ecs-gpu-init/cmd/ecs-gpu-init/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/NVIDIA/go-nvml/pkg/nvml"
+)
+
+const (
+	// NvidiaGPUInfoFilePath is the file path where GPUs and driver info are saved
+	NvidiaGPUInfoFilePath = "/var/lib/ecs/gpu/nvidia-gpu-info.json"
+)
+
+func main() {
+	if err := generateGPUInfoConfig(); err != nil {
+		panic(err)
```

Comment:
Is panic really the right function here?

Reply:
> Is panic really the right function here?

I suppose you could print the error and exit 1 instead of spewing a stack trace that points to main.